Dockerfile.base

```diff
@@ -48,8 +48,8 @@ RUN apt-get update \
 	&& chmod 0440 /etc/sudoers.d/$user \
 	&& chown $UID /calypso \
 	&& rm -rf /var/lib/apt/lists/* \
-	&& apt autoremove --purge \
-	&& apt autoclean
+	&& apt-get autoremove --purge \
```

Comment:
Explanation: these are ancillary changes. It arose from the warning I saw that `apt does not have a stable CLI interface`. 

See [this post](https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning) for more info.